WordPress/wordpress-playground
# Add/allow import site gutenberg pr

## Motivation for the change, related issues

This is related to supporting visual regression testing of Gutenberg PRs with Playground.
https://github.com/WordPress/gutenberg/pull/62729

Importing content into a Playground makes testing easier. This will allow the supported `import-site` parameter to be used on this page as well.

## Implementation details

This change adds a check for a the `import-site` query parameter and adds a step to the blueprint when it is found. 

The link has minimal validation, but I don't believe this creates a significant security concern. In theory someone could corrupt the blueprint by directing people to this URL with a malformed query parameter. 

Validating that the parameter is a URL, and not passing the query string or hash seemed to mitigate the most obvious offenses.

## Testing Instructions (or ideally a Blueprint)

 1. Run locally 
 2. Visit `gutenberg.html` with a query parameter `import-site` pointing to a valid Playground ZIP import.
 3. Confirm that Playground has imported content from ZIP file.
